Assets/Scripts/UI/Menus/AudioOptionsMenu.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+using UnityEngine.UI;
+
+namespace OperationBlackwell.UI {
+	public class AudioOptionsMenu : Menu {
+
+		[SerializeField] private Slider backgroundMusicSlider_;
+		[SerializeField] private Slider soundEffectSlider_;
+		[SerializeField] private AudioMixer backgroundMusicMixer_;
+		[SerializeField] private AudioMixer soundEffectMixer_;
+
+		private void Start() {
+			UpdateSliders();
+		}
+
+		public override void ReturnOrExit() {
+			audioOptionsMenu_.SetActive(false);
+			optionsMenu_.SetActive(true);
+		}
+
+		private void UpdateSliders() {
+			float backgroundMusicVolume;
+			backgroundMusicMixer_.GetFloat("MusicVolume", out backgroundMusicVolume);
+			backgroundMusicSlider_.value = Mathf.Pow(10, backgroundMusicVolume / 20);
+
+			float soundEffectVolume;
+			soundEffectMixer_.GetFloat("MusicVolume", out soundEffectVolume);
```

Comment:
What is this string in this function `MusicVolume`? And why is it the same for this and the background music slider?

Reply:
It's the name of the exposed variable from the editor. As both sliders refer to their own mixer, having them have the same name isn't an issue.